client/src/components/ResourceHub/EditResourceModal.tsx

```diff
@@ -0,0 +1,145 @@
+import React, { ReactElement, FC, Dispatch, useEffect } from 'react';
+import { Modal, Button, Form, Input, Checkbox } from 'semantic-ui-react';
+import { IResource } from 'ssw-common';
+
+import { editResource, isError } from '../../utils/apiWrapper';
+
+import '../../css/EditResourceModal.css';
+
+const teams = [
+  'General',
+  'Editing',
+  'Factchecking',
+  'Illustration',
+  'Photography',
+  'Onboarding',
+  'Visuals',
+  'Writing',
+];
+
+interface IProps {
+  isOpen: boolean;
+  resource: IResource | null;
```

Comment:
Idk how others feel about this but rather than declaring resource to be potentially null, we can say

`resource?: IResource`

this just states that the resource could be undefined